iOS/Collection View Management /CollectionViewDataSource.swift

```diff
@@ -62,17 +63,23 @@ class CollectionViewDataSource<SectionType: Hashable, ItemType: Hashable> {
 // These functions just forward to the corresponding functions to the underlying NSDiffableDataSource
 extension CollectionViewDataSource {
 
-    func apply(_ snapshot: NSDiffableDataSourceSnapshot<SectionType, ItemType>,
+    // MARK: - Standard DataSource Functions
+
+    func apply(_ snapshot: SnapshotType, animatingDifferences: Bool = true) {
+        self.diffableDataSource.apply(snapshot, animatingDifferences: animatingDifferences, completion: nil)
+    }
+
+    func apply(_ snapshot: SnapshotType,
```

Comment:
Made a synchronous version of apply. Sometimes you don't have to worry about doing follow up work. With this function you can just fire and forget.